language/move-stdlib/tests/move_unit_test.rs

```diff
@@ -1,12 +1,38 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use move_stdlib::move_stdlib_named_addresses;
-use move_unit_test::UnitTestingConfig;
+use move_cli::package::cli;
+use move_core_types::account_address::AccountAddress;
+use move_stdlib::{natives::all_natives, path_in_crate};
+use tempfile::tempdir;
 
-move_unit_test::register_move_unit_tests!(
-    UnitTestingConfig::default_with_bound(Some(100_000))
-        .with_named_addresses(move_stdlib_named_addresses()),
-    ".",
-    r".*\.move$"
-);
+fn run_tests_for_pkg(path_to_pkg: impl Into<String>) {
```

Comment:
Switched these tests over to use the package system so we can use a tempdir -- observed some intermittent failures before the switch